### ISSUE TITLE
Error when trying to predict in Text Classification with a custom model

### ISSUE THREAD

**Issue by nahueltori** [keras-team/autokeras#632]
### Bug Description
When trying to predict in my custom model, the following error is thrown:

<code>
Traceback (most recent call last):
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/venv/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/venv/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/venv/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/venv/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/venv/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/venv/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/venv/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/main.py", line 104, in add_clasificacion
    transaccion.clasificar()
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/tagger/model/transaccion.py", line 40, in clasificar
    prediccion = get_prediction(self.texto)
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/tagger/prediccion_autokeras.py", line 16, in get_prediction
    result = txt_clf.predict(content)
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/autokeras/text/text_supervised.py", line 92, in predict
    num_labels=self.num_labels)
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/autokeras/text/pretrained_bert/modeling.py", line 444, in from_pretrained
    model = cls(config, *inputs, **kwargs)
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/autokeras/text/pretrained_bert/modeling.py", line 608, in __init__
    self.classifier = nn.Linear(config.hidden_size, num_labels)
  File "/home/nahuel/PycharmProjects/digitaldocs-tagger/venv/lib/python3.6/site-packages/torch/nn/modules/linear.py", line 51, in __init__
    self.weight = Parameter(torch.Tensor(out_features, in_features))
TypeError: new() received an invalid combination of arguments - got (NoneType, int), but expected one of:
 * (torch.device device)
 * (torch.Storage storage)
 * (Tensor other)
 * (tuple of ints size, torch.device device)
      didn't match because some of the arguments have invalid types: (NoneType, int)
 * (object data, torch.device device)
      didn't match because some of the arguments have invalid types: (NoneType, int)
</code>

### Reproducing Steps
  * Step 1: Trained my model with a custom dataset, 777 records were given.
  * Step 2: Predict using TextClassifier.predict.

### Expected Behavior
Return the prediction on the model.

### Setup Details
Include the details about the versions of:
 - OS type and version: Ubuntu 16.04
 - Python: 3.6
 - autokeras: 0.4.0
 - scikit-learn: 0.20.2
 - numpy: 1.16.1
 - keras: 2.2.4
 - scipy: 1.2.0
 - tensorflow: 1.13.1
 - pytorch: 1.0.1.post2

### Additional context
I'm using a custom Text Classification dataset with only 1 characteristic, and 10 different labels for classify it.

Please ask for any other details you may need.

Thank you very much.

**Comment by nahueltori** [comment]
Checking the error message in detail, the problem seems to be in the variable: `self.num_labels`.
Unlinke the example in `https://github.com/keras-team/autokeras/blob/master/examples/task_modules/text/text_classification.py`, in my code I'm training my model in one function, and predicting results in another.
I create the object from the class TextClassifier once for the training, and a new object in the predict function. When debbuging the predict function, I have found that self.num_labels is always `None`, and that is the value sent in the "Prepare model" `BertForSupervisedTasks.from_pretrained`.
I tested adding the `num_labels` parameter to the TextClassifier class, and it worked ok.
Hope this helps to solve the issue!
Regards.
